stdlib/public/SDK/CoreImage/CoreImage.swift

```diff
@@ -50,9 +50,9 @@ extension CIFilter {
 extension CISampler {
   // - (id)initWithImage:(CIImage *)im keysAndValues:key0, ...;
   convenience init(im: CIImage, elements: (String, Any)...) {
-    var dict: [NSObject : Any] = [:]
+    var dict: [AnyHashable : Any] = [:]
```

Comment:
While you are changing this code, could you reserve capacity in the new dictionary?